### PR TITLE
Updates versions of dependencies to latest

### PR DESCRIPTION
**Pull request by ostreifel**

### CODE REVIEW

File: scripts/app.ts [hunk 1]
```diff
@@ -19,5 +19,4 @@ var actionProvider =  {
 };
 
 // Register context menu action provider
-const publisherId = VSS.getExtensionContext().publisherId;
-VSS.register(`${ publisherId }.vsts-extension-ts-seed-simple.vsts-extension-ts-seed-simple-action`, actionProvider);
\ No newline at end of file
+VSS.register(VSS.getExtensionContext().extensionId, actionProvider);
```
Comment (cschleiden) [inline scripts/app.ts]: Is extensionId fully qualified?

Comment (ostreifel) [reply scripts/app.ts]: Updated to contribution id.

File: scripts/app.ts [hunk 2]
```diff
@@ -19,5 +19,4 @@ var actionProvider =  {
 };
 
 // Register context menu action provider
-const publisherId = VSS.getExtensionContext().publisherId;
-VSS.register(`${ publisherId }.vsts-extension-ts-seed-simple.vsts-extension-ts-seed-simple-action`, actionProvider);
\ No newline at end of file
+VSS.register(VSS.getContribution().id, actionProvider);
```
Comment (cschleiden) [inline scripts/app.ts:22]: this is fully qualified?

**Review by cschleiden — APPROVED** [review]

File: scripts/app.ts (hunk 2 above)
Comment (cschleiden) [reply scripts/app.ts:22]: @ostreifel not sure how notifications work here ;) mentioning just in case

Comment (ostreifel) [reply scripts/app.ts:22]: Yes. I used this notation for board control and pull request search.

**Pull request merged**